corehq/util/datadog/utils.py

```diff
@@ -87,3 +90,8 @@ def get_url_group(url):
         return parts[3] if len(parts) >= 4 else default
 
     return default
+
+
+def update_datadog_metrics(metrics={}):
```

Comment:
`metrics` shouldn't have a default value (and in general you shouldn't use mutable types as default values: see http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments)